docs/first-team.mdx

```diff
@@ -0,0 +1,32 @@
+---
+title: Add your first Team
+---
+
+Much like services, adding teams is done by defining them via the GitOps workflow.  The simplest way to do this would be to create a `teamname.effx.yaml` file in a Git repo where it makes the most sense.
+
+Some of our users with monorepos will create a `teams` directory at the top-level of their repo and add a `teamname.effx.yaml` for each team within.  You can also create a `teams.effx.yaml` and define all of your teams in a single file as we support multiple objects per `effx.yaml` file.
+
+## Adding your first team
```

Comment:
This is another redundant section. I think it's better to re-use the Resource Types UI than restate it here.